tests/qinfo_tools/test_qfi.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import annotations
+
+import random
+
+import numpy as np
+import torch
+from qadence import BasisSet, QuantumCircuit
+from qadence.constructors import feature_map, hea
+from qadence.operations import RX
+from torch import Size, allclose
+
+from qadence_libs.qinfo_tools import get_quantum_fisher, get_quantum_fisher_spsa
+
+SEED = 42
+torch.manual_seed(SEED)
+np.random.seed(SEED)
+random.seed(SEED)
+
+
+def create_hea_circuit(n_qubits: int, layers: int) -> QuantumCircuit:
```

Comment:
Maybe you can use [this](https://github.com/pasqal-io/qadence/blob/a92780f75da81bce26f83a99fd5b46542531cdff/qadence/constructors/ansatze.py#L17) ?

Reply:
I'm using it inside the function, but `hea()` outputs an `AbstractBlock` with no FM and basically what I do inside the function is add a FM and wrap everything in a `QuantumCircuit` since that's what I need